cognite/client/data_classes/ai.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any
+
+from cognite.client.data_classes.data_modeling import NodeId
+
+
+class AnswerLanguage(Enum):
+    Chinese = "Chinese"
+    Dutch = "Dutch"
+    English = "English"
+    French = "French"
+    German = "German"
+    Italian = "Italian"
+    Japanese = "Japanese"
+    Korean = "Korean"
+    Latvian = "Latvian"
+    Norwegian = "Norwegian"
+    Portuguese = "Portuguese"
+    Spanish = "Spanish"
+    Swedish = "Swedish"
+
+
+@dataclass
+class Summary:
```

Comment:
Please inherit from `CogniteObject`. It will add serialize/deserialize methods and add automatic testing of the data class to future-proof it.

For resources that you'd like to visualize nicely in e.g. notebooks, consider inheriting from `CogniteResource` which also adds a `to_pandas` method and `_repr_html_`